versions/development/SPEC.md

```diff
@@ -2896,9 +2896,15 @@ Note that because some call inputs are left unsatisfied, this workflow could not
 
 Once workflow inputs are computed (see previous section), the value for each of the fully-qualified names needs to be specified per invocation of the workflow. The format of workflow inputs is implementation specific.
 
-### Cromwell-style Inputs
+### Common WDL JSON Input Format
 
-The "Cromwell-style" input format is widely supported by WDL implementations and recommended for portability purposes. In the Cromwell-style format, workflow inputs are specified as key/value pairs in JSON or YAML. The mapping to WDL values is codified in the [serialization of task inputs](#serialization-of-task-inputs) section.
+The Common WDL JSON input format is an input format that is required to be 
+accepted by any workflow engine. It is the input format that is used for 
+performing language compliance tests on the engine.
+
+In this common format, workflow inputs are 
```

Comment:
This should be usable for both workflow and task inputs/outputs.

Reply:
This is in the workflow inputs chapter. Can you clarify what you mean by task inputs in this context?

Reply:
Some workflow engines (Cromwell, dxWDL) support running individual tasks in addition to workflows. This input specification should be the same for both workflows and inputs, and it should be required for any workflow engine that allows running tasks.

Reply:
I think this is good. My only comment is to consider shortening the concept name from `Common WDL JSON Input Format` to `JSON Input Format`. It makes it more readable, and reduces potential confusion with the "Common Workflow Language". After all, this document **is** the authoritative document on the WDL language, so the words `common` and `WDL` should be superfluous. Any changes an engine makes are not standard/common by definition.

Reply:
@orodeh Very good suggestion. It was also a bit tedious to type. 

@jdidion, running individual tasks is a feature that is not listed in the spec. So workflow engines can do as they please with regards to running tasks. We can of course make running tasks part of the spec, but that will be another PR entirely.